featuros-plugin/src/main/java/io/fourfinanceit/featuros/Featuros.java

```diff
@@ -0,0 +1,66 @@
+package io.fourfinanceit.featuros;
+
+import jodd.http.HttpException;
+import jodd.http.HttpResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+import java.util.Optional;
+
+import static jodd.http.HttpRequest.post;
+
+/**
+ * Plugin that reaches out to featuros dashboard and notifies of current deployment.
+ * When not using a boot-starter module, it has to be called manually, e.g.:
+ * <pre>
+ *     public static void main(String... args) {
+ *         SpringApplication.run(args);
+ *         Featuros.notification();
+ *     }
+ * </pre>
+ */
+public class Featuros {
+
+    private static final Logger log = LoggerFactory.getLogger(Featuros.class);
```

Comment:
Having a groovy style logger with lombok would be nice, but we don't really want to throw in a 1MB lib unless we have more use of it than this case.